subxt/src/config/mod.rs

```diff
@@ -82,7 +82,7 @@ pub trait Hasher {
 }
 
 /// This represents the block header type used by a node.
-pub trait Header: Sized + Encode {
+pub trait Header: Sized + Encode + Decode {
```

Comment:
why is `Decode` needed now?

Reply:
In the old API methods, header came back as JSON, but in the new one it comes back as hex encoded SCALE encoded bytes instead :)